conduct.md

```diff
@@ -11,7 +11,39 @@
 * Attendance
 * Deadlines
 * Communication
-* Participation
+
+### Participation
+
+Every team member needs to be actively and cooperatively involved with the
+projects at hand. This will ensure the flow of questions, ideas, and thoughts
+in a respectful manner. Our project at its root includes participation, candidness
+and insightful collaboration. Participating in project discussion means engaging
+in conversation, not just answering questions. The goal here is to show up and
+truly be all there.
+
+* **Acceptable Conduct (A):**
+  * *Members shall*
+    * Must contribute when designated to, in a timely manner
+    * Quick to respond in Slack to clarify confusing issues or assist other colleagues
+    * Everyone’s opinion must be considered
+    * Willing to meetup and work outside of class
+    * Approach issues and team members with professionalism
+    * Demonstrate continuous engagement with tasks
+    * Open minded
+    * Raise a disagreeing opinion in a respectful manner
+
+* **Unacceptable Conduct (U):**
+  * *Members shall not*
+    * Demonstration of disrespect for team members
+    * Having an inappropriate professional boundary with team members
+    * Poor participation and attitude to their lab delegated tasks
+    * Patterns of poor communication or little effort for
+    collaboration in lab and Slack
+    * Disrupt working groups’ effectiveness by lack of collaboration
+    * Lack of insight into his or her level of competence
+    * Attempt to dominate a lab discussions
+    * No show in lab sessions without the consent of the professor
```

Comment:
This section also should be rephrased as above. Each bullet point should connect to "Member shall not"